src/getCurrentAWSRegion.ts

```diff
@@ -5,7 +5,9 @@ import {AWSRegion} from './aws-regions';
 function getCurrentAWSRegion(): AWSRegion {
   // MUST be called after configureAWS()
   // this cast is only safe after that
-  const region = (aws.config.region || process.env.AWS_REGION) as AWSRegion; // tslint:disable-line
+  const region = (aws.config.region ||
+                  process.env.AWS_REGION ||
+                  process.env.AWS_DEFAULT_REGION) as AWSRegion; // tslint:disable-line
```

Comment:
Can `AWS_DEFAULT_REGION` be absent? Should there be a check and `throw` if it fails?